bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -61,6 +61,11 @@ <h2 class="heading">
           <a href="{{ url('firefox.sms') }}">Text a download link to my Android phone</a>
         </p>
       </div>
+
+      {% if l10n_has_tag('ios_announce') %}
```

Comment:
Let's use firefox/new to drive this string (longer explanation in the main discussion)
```
if l10n_has_tag('ios_announce', 'firefox/new')
``